packages/core/src/transport/flushController.spec.ts

```diff
@@ -85,25 +85,25 @@ describe('flushController', () => {
     it('notifies when the session expires', () => {
       flushController.notifyBeforeAddMessage(SMALL_MESSAGE_BYTE_COUNT)
       flushController.notifyAfterAddMessage()
-      sessionExpireObservable.notify()
+      sessionExpireObservable.notify('')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about using a type constant instead of an empty string?